src/Entities/Pharmacies.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
```

Comment:
remember to only include the entities that are used in this PR (it seems like there are more than the last PR)

Reply:
I know what happened is that I was going to use them but got pressed for time and fell back to the simple stuff to complete the module. These will be used in the future. Removed for now.

Reply:
I put it back because it is used in the SetPharmacyController line 13